charts/vsphere-csi/v2.1.0/values.yaml

```diff
@@ -1,29 +1,65 @@
 vCenter:
   host: ""
-  datacenters: ""
   port: 443
   insecureFlag: true
+  clusterId: ""
+  datacenters: ""
   username: ""
   password: ""
   configSecret:
     name: "vsphere-config-secret"
     generate: true
-    configTemplate: |
-      [Global]
-      cluster-id = {{ uuidv4 | quote }}
-      user = {{ .Values.vCenter.username | quote }}
-      password = {{ .Values.vCenter.password | quote }}
-      port = {{ .Values.vCenter.port | quote }}
-      insecure-flag = {{ .Values.vCenter.insecureFlag | quote }}
+  configTemplate: |
+    [Global]
+    cluster-id = {{ required ".Values.vCenter.clusterId must be provided" (default .Values.vCenter.clusterId .Values.global.cattle.clusterId) | quote }}
+    user = {{ .Values.vCenter.username | quote }}
+    password = {{ .Values.vCenter.password | quote }}
+    port = {{ .Values.vCenter.port | quote }}
+    insecure-flag = {{ .Values.vCenter.insecureFlag | quote }}
 
-      [VirtualCenter {{ .Values.vCenter.host | quote }}]
-      datacenters = {{ .Values.vCenter.datacenters | quote }}
+    [VirtualCenter {{ .Values.vCenter.host | quote }}]
+    datacenters = {{ .Values.vCenter.datacenters | quote }}
+
+csiController:
+  csiResizer:
+    enabled: false
+  image:
+    repository: gcr.io/cloud-provider-vsphere/csi/release/driver
+    tag: v2.1.0
+    pullPolicy: Always
+    csiAttacher:
+      repository: quay.io/k8scsi/csi-attacher
+      tag: v3.0.0
+    csiResizer:
+      repository: quay.io/k8scsi/csi-resizer
+      tag: v1.0.0
+    livenessProbe:
+      repository: quay.io/k8scsi/livenessprobe
+      tag: v2.1.0
+    vsphereSyncer:
+      repository: gcr.io/cloud-provider-vsphere/csi/release/syncer
+      tag: v2.1.0
+      pullPolicy: Always
+    csiProvisioner:
+      repository: quay.io/k8scsi/csi-provisioner
+      tag: v2.0.0
+  nodeSelector: {}
 
 csiMigration:
   enabled: false
 
-csiResizer:
-  enabled: false
+csiNode:
+  prefixPath: ""
+  image:
+    repository: gcr.io/cloud-provider-vsphere/csi/release/driver
+    tag: v2.1.0
+    pullPolicy: Always
+    nodeDriverRegistrar:
+      repository: quay.io/k8scsi/csi-node-driver-registrar
+      tag: v2.0.1
+    livenessProbe:
+      repository: quay.io/k8scsi/livenessprobe
+      tag: v2.1.0
```

Comment:
Seems odd to have these repo's and tags under a different image node, but if this is way upstream did it I guess leave it. Sure this isn't a bad indent right ?

Reply:
They are set based on the workload E.g images used by containers in vsphere-csi-node vs vsphere-csi-controller.